ghost/oembed-service/lib/oembed-service.js

```diff
@@ -184,12 +171,9 @@ class OEmbed {
      * @returns {Promise<{url: string, body: Object}>}
      */
     async fetchPageJson(url) {
-        const {body, url: pageUrl} = await this.fetchPage(
-            url,
-            {
-                json: true
-            });
-
+        const res = await this.fetchPage(url, {}); // .json() doesn't seem to parse appropriately... bug with got v10?
+        const body = JSON.parse(res.body);
+        const pageUrl = res.url;
```

Comment:
```suggestion
        const res = await this.fetchPage(url, {responseType: 'json'});
        const body = res.body;
        const pageUrl = res.url;
```
Gonna check this

Reply:
I think that should work, because .json() automatically returns the body instead of the response